examples/eight_schools/eight_schools.py

```diff
@@ -0,0 +1,94 @@
+"""implement the stan 8 schools example
+using the recommended non-centred parameterisation
+
+https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started
+http://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html
+we slightly modify the stan example to:
+                                         avoid improper priors
+                                         avoid half-cauchy priors
+make the statement of the model directly comparable
+
+This model has a hierachy and an inferred variance - yet the example is
+very simple - only the Normal distribution is used"""
```

Comment:
Last docstring `"""` should be at newline. All sentences should read like English (e.g., periods; grammatical sentences).